.github/workflows/ci.yml

```diff
@@ -108,9 +110,43 @@ jobs:
           cache-from: type=registry,ref=${{ env.DOCKER_IMAGE_TAG }}
           cache-to: type=inline
 
+  docker-publish-release:
+    if: (github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/'))
```

Comment:
From the Safe Config Service:

```
if: (github.event_name == 'release' && github.event.action == 'released')
```

It's a better match in my opinion given the event that triggered it.

https://github.com/safe-global/safe-config-service/blob/3877fdc19a9e3109b0f7515025d9c3cecf099c58/.github/workflows/test.yml#L190C5-L190C78

Reply:
Absolutely! Thanks.

Changed in[374283a](https://github.com/safe-global/safe-client-gateway-nest/pull/525/commits/374283a78f1744be50dad3197fcaa262d6539a09)